db_management.py

```diff
@@ -1,31 +1,52 @@
-import sqlite3
+from sqlalchemy import create_engine, Column, Integer, String
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.ext.declarative import declarative_base
 
 
-def new_user_creation(username_text, pass_text):
-    conn = sqlite3.connect('data_base.db')
-    curs = conn.cursor()
-    params = (username_text, pass_text)
-    curs.execute("INSERT INTO users VALUES (NULL, ?, ?)", params)
-    conn.commit()
-    conn.close()
+engine = create_engine('sqlite:///data_base.db')
+conn = engine.connect()
+Session = sessionmaker(bind=engine)
+session = Session()
+Base = declarative_base()
+
+
+class User(Base):
+    __tablename__ = 'users'
+    id = Column(Integer, primary_key=True)
+    Username = Column(String, nullable=False, unique=True)
+    Password = Column(String, nullable=False, unique=True)
+
+
+def create_connection():
+    Base.metadata.create_all(engine)
+
+
+def check_if_user_exists(str_username, str_pass):
+    Session = sessionmaker(bind=engine)
```

Comment:
I think it's better to have a method in db_management.py to create a connection and use it here. You don't want to write these lines in every place you need a connection to the DB, rght?

Reply:
What exactly did you fix?